pkgs/merkle-tree-wasm/src/lib.rs

```diff
@@ -20,7 +20,7 @@ fn internal_generate_merkle_proof<F: PrimeField>(
 
     let mut padded_leaves = leaves.clone();
     // Pad the leaves to equal the size of the tree
-    padded_leaves.resize(1 << depth, "0".to_string());
+    padded_leaves.resize(1 << depth, "0x".to_string());
```

Comment:
@0xisk Was using `"0"` deliberate?

I am getting 
```
panicked at pkgs/merkle-tree-wasm/src/lib.rs:33:60:
called `Result::unwrap()` on an `Err` value: OddLength
```

When I am trying to build a merkle proof otherwise.

Reply:
I think the merkle-tree-wasm crate could benefit from a more robust error handling.
You `unwrap` lot of values but never deal with the errors.

Reply:
> I think the merkle-tree-wasm crate could benefit from a more robust error handling. You `unwrap` lot of values but never deal with the errors.

Yes that is fixed in this PR: #380

Reply:
> @0xisk Was using `"0"` deliberate?
> 
> I am getting
> 
> ```
> panicked at pkgs/merkle-tree-wasm/src/lib.rs:33:60:
> called `Result::unwrap()` on an `Err` value: OddLength
> ```
> 
> When I am trying to build a merkle proof otherwise.

I dont think the error you are getting is from this function, it just makes sure to have leaves number equals to 2^depth, and it fills the rest with "0". 
Maybe we should handle an error back to the user if you input a big anonset exceeds the allowed max no of members.

Reply:
@0xisk Please have a look at https://github.com/anonklub/anonklub/issues/381. You should be able to reproduce it yourself by checking out the `merkle-tree-bug` branch.
It looks like your adding leaf with `0` (indeed to fill the rest).
But then you are doing `.replace("0x", "")`... and it can't find `0x` because it is only `0`.
I wonder why this error is not happening in the ui with the wasm code.

Reply:
@0xisk I thought your previous PR had fixed this.
And I forgot I still had this change on my branch.
If I remove it, I still get the same error.